dbt_project/models/CLEANED/schema.yml

```diff
@@ -38,6 +42,11 @@ models:
       - name: "company"
         description: "The name of the company that this user belongs to."
         data_type: "str"
+        tests:
+          - dbt_expectations.expect_column_values_to_match_like_pattern_list:
+              like_pattern_list: ["%Sport%","%Co%","%Ltd%", "%Shop%"]
+              match_on: any # (Optional. Default is 'any', which applies an 'OR' for each pattern. If 'all', it applies an 'AND' for each regex.)
+              alias: example_alias
```

Comment:
I take it the alias didn't work?

Reply:
I have uncommitted changes for how to do it with dbt models and asset names, but yeah it looks like it would require changes to the Dagster-dbt library for test result aliases. 

Same story for test result metadata, though I think we could get it by accessing the raw_event_stream and then appending the output to the AssetCheckResult